### PR TITLE
admin-page

### PR DESCRIPTION
**Pull request by okso-hub**
**created admin page with links to (marked ones are working)**
- [x] managing cinema halls
- [x] managing showtimes
- [ ] managing reservations

**movies page only shows movies which have showtimes**

> [!important]
> currently, as there is no authentification for the administrator, you can only access the page by adding /admin to the url.

### CODE REVIEW

File: src/routes/admin/+page.svelte
Comment (okso-hub) [inline src/routes/admin/+page.svelte:1]: -- multiple style tags (extra one in main)

File: src/routes/admin/manage-halls/create-hall/+page.svelte
Comment (okso-hub) [inline src/routes/admin/manage-halls/create-hall/+page.svelte:1]: i think there are some unused functions.

File: src/routes/admin/manage-screenings/+page.server.ts
Comment (okso-hub) [inline src/routes/admin/manage-screenings/+page.server.ts:1]: request for future: show movie title, not omdb_id

File: src/routes/admin/manage-screenings/+page.svelte
Comment (okso-hub) [inline src/routes/admin/manage-screenings/+page.svelte:1]: request for future: show movie title, not omdb_id

**Review by okso-hub — COMMENTED** [review]
overall good, but some improvements could be made in:
- src/routes/admin/+page.svelte
- src/routes/admin/manage-halls/create-hall/+page.svelte
comments made on the files

File: src/routes/admin/manage-halls/create-hall/+page.svelte [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+		row_count = 0;
+		col_count = 0;
+		rows.update(() => []);
+	}
+
+	function preview() {
+		rows.set(Array.from({ length: row_count }, () => Array(col_count).fill('Regular')));
+	}
+
+	async function safe() {
```
Comment (raphaelplt) [inline src/routes/admin/manage-halls/create-hall/+page.svelte:41]: `async function save() {`

**Review by raphaelplt — CHANGES_REQUESTED** [review]
change character from "safe" to "save"

**Review by raphaelplt — APPROVED** [review]

**Pull request merged**